config/tracing.go

```diff
@@ -0,0 +1,46 @@
+package config
+
+import (
+	"errors"
+)
+
+type TracingConfig struct {
```

Comment:
- adding `Enabled` would be more straightforward.

Reply:
done